torch/csrc/utils/python_strings.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <Python.h>
+#include <stdexcept>
+#include "object_ptr.h"
+
+// Utilities for handling Python strings. Note that PyString, when defined, is
+// the same as PyBytes.
+
+// Returns true if obj is a bytes/str or unicode object
+// In existing CPython implementations, this does not require the GIL
```

Comment:
Do we depend on that anywhere or is it just a note? Also, it'd be better to say "as of 3.6" instead of "in existing", because it can change any moment.